doc/en/weechat_dev.en.adoc

```diff
@@ -1362,5 +1362,5 @@ warnings, ... These words must be kept unchanged:
 When there is a name after `+<<link_name>>+`, then you must translate it:
```

Comment:
I know that `must` is used in the instructions, but it's fairly unhelpful.

There will be translations where a thing is translated to itself and it's meaningless to insist that it must be translated. It makes even less sense later.